MoreMenu.old.js

```diff
@@ -388,7 +388,7 @@
 					namespaceRestrict : [ 0, 2, 118 ]
 				},
 				'Transclusion count' : {
-					url : '//tools.wmflabs.org/templatecount/index.php?lang=' + contentLanguage + '&name=' + encodeURIComponent( mw.config.get( 'wgTitle' ) ) + '&namespace=' + namespaceNumber,
+					url : 'https://templatecount.toolforge.org/index.php?lang=' + contentLanguage + '&name=' + encodeURIComponent( mw.config.get( 'wgTitle' ) ) + '&namespace=' + namespaceNumber,
 					namespaceRestrict : [2, 4, 5, 10, 11, 12, 13, 100, 101, 828],
 					noticeProjectRestrict : [ 'wikipedia' ]
 				}
```

Comment:
This file is just a copy of an older version of the gadget and isn't used anymore. Sorry, I should remove it.